src/jemalloc.c

```diff
@@ -1713,8 +1713,7 @@ typedef struct static_opts_s static_opts_t;
 struct static_opts_s {
 	/* Whether or not allocation size may overflow. */
 	bool may_overflow;
-	/* Whether or not allocations of size 0 should be treated as size 1. */
```

Comment:
Wait, this pull request adds implicit support for `mallocx(0, 0)`?  If so, I don't think we should go down this path.